src/Products.jl

```diff
@@ -178,6 +178,8 @@ function locate(lp::LibraryProduct; verbose::Bool = false,
 
                     if verbose
                         @info("$(dl_path) cannot be dlopen'ed")
```

Comment:
Let's have `res` be a second parameter to the above `@info` call.

Reply:
done, with new try catch